models/index.js

```diff
@@ -0,0 +1,24 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+    class User extends Model {
+      static associate(models) {
+      }
+    }}
+
+    User.init(
+        {
```

Comment:
The init should be structured from 
{
   username: DataTypes.STRING,
   email: DataTypes.STRING,
   etc...
}

It should be
username {
   type: DataTypes.STRING,
   allowNull: false,
},
email {
   type: DataTypes.STRING,
   allowNull: false,
},
etc...

For more reference on syntax check class work `14-MVC/01-Activities/23-Ins_Auth-Review/models/User.js`